server/src-lib/Hasura/RQL/Types/ScheduledTrigger.hs

```diff
@@ -0,0 +1,103 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
Please remove this if it’s no longer needed!

Reply:
Still need `FromJSON` and `ToJSON`. Should I put it someplace else?

Reply:
Yes, I would move those into `Hasura.RQL.Instances` as well. I think it’s a good idea to keep all orphans in one module. If that module gets too big, we can break it up into separate modules and have one module that re-exports all of them, but I don’t think that’s a problem yet.

The reason it’s good to keep orphan instances in one module has to do with the way GHC does instance coherence checking. If you have two modules, `A` and `B`, that each declare conflicting instances, GHC won’t report an error if neither module depends on the other. If you have a third module `C` that depends on both `A` and `B`, then GHC will report an error if `C` attempts to use either instance… but if `C` never needs that instance, no error will be reported.

For that reason, it’s a good idea to keep all your orphan instances in one module. That way you can’t ever have this “conflicting instances declared in two different places” problem.